examples/features/authz/server/main.go

```diff
@@ -197,13 +201,30 @@ func main() {
 		log.Fatalf("Loading credentials: %v", err)
 	}
 
-	// Create an authorization interceptor using a static policy.
-	staticInteceptor, err := authz.NewStatic(authzPolicy)
-	if err != nil {
-		log.Fatalf("Creating a static authz interceptor: %v", err)
+	// Create authorization interceptors according to the authz-option command-line flag.
+	var unaryAuthzInt grpc.UnaryServerInterceptor
+	var streamAuthzInt grpc.StreamServerInterceptor
+	switch *authzOpt {
+	case authzOptStatic:
+		// Create an authorization interceptor using a static policy.
+		staticInt, err := authz.NewStatic(authzPolicy)
```

Comment:
Prefer the full name `Interceptor` rather than shortening to `Int`, please change throughout

Reply:
Updated.